.github/workflows/tests.yml

```diff
@@ -133,7 +133,7 @@ jobs:
           if [[ -n "${{ matrix.conda-extra-pkgs }}" ]]; then
             conda install "${{ matrix.conda-extra-pkgs }}"
           fi
-          python -m pip install -e .[$QUTIP_TARGET]
+          python -m pip install -e .[$QUTIP_TARGET] -vv
```

Comment:
Should we remove the `-vv` now that the tests pass?

Reply:
```suggestion
          python -m pip install -e .[$QUTIP_TARGET]
```